src/Runner.Common/Terminal.cs

```diff
@@ -16,6 +16,7 @@ public interface ITerminal : IRunnerService, IDisposable
         bool Silent { get; set; }
         string ReadLine();
         string ReadSecret();
+        void StdOutWriteLine(string line);
```

Comment:
Let's add an extra-parameter to `void WriteLine(string line, ConsoleColor? colorCode = null, bool skipTracing = false);`


```
public void WriteLine(string line, ConsoleColor? colorCode = null, bool skipTracing = false)
        {
            if (!skipTracing)
            {
                Trace.Info($"WRITE LINE: {line}");
            }
            if (!Silent)
            {
                if (colorCode != null)
                {
                    Console.ForegroundColor = colorCode.Value;
                    Console.WriteLine(line);
                    Console.ResetColor();
                }
                else
                {
                    Console.WriteLine(line);
                }
            }
        }
```